apps/src/applab/setPropertyDropdown.js

```diff
@@ -174,19 +174,28 @@ Object.keys(PROP_NAMES).map(function (elementType) {
   });
 });
 
-// May belong in droplet
-function getValueOfNthParam(block, n) {
+/**
+ * @param {DropletBlock} block
+ * @param {AceEditor}
+ */
+function getFirstSetPropertyParam(block, editor) {
+  if (!block) {
+    // If we're not given a block, assume that we're in text mode
+    var cursor = editor.session.selection.getCursor();
+    var contents = editor.session.getLine(cursor.row).substring(0, cursor.column);
+
+    var match = /setProperty\(['|"](.*)['|"]/.exec(contents);
```

Comment:
I'm not sure this regex is quite right.  `['|"]` matches the set of characters `'`, `|`, & `"`.